src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is started working");

```

Comment:
The message 'Excavator is started working' could be more informative by specifying that it is an Excavator which is working, as per the checklist. Consider rephrasing to 'Excavator started its work' to make it clear and consistent with the task requirements.